app/javascript/components/request-workflow-status/data.js

```diff
@@ -43,13 +43,43 @@ const convertDate = (date) => {
   return formattedDate.toString();
 };
 
+// Converts the duration time in ms and returns a string in format: w days x hours y minutes z seconds
+// duration: time in ms
+const convertDuration = (duration) => {
+  const durationString = moment.duration(duration, 'milliseconds').toISOString().split('PT')[1];
+  let startIndex = 0;
+  let resultString = '';
+  if (durationString.indexOf('H') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('H') + 1)} `;
+    startIndex = durationString.indexOf('H') + 1;
+  }
+  if (durationString.indexOf('M') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('M') + 1)} `;
+    startIndex = durationString.indexOf('M') + 1;
+  }
+  if (durationString.indexOf('S') >= 0) {
+    resultString += `${durationString.slice(startIndex, durationString.indexOf('S') + 1)} `;
+    startIndex = durationString.indexOf('S') + 1;
+  }
+  return resultString;
```

Comment:
As discussed, would it be possible to lowercase the string at the end?  I think `1h 45m 3s` looks better than `1H 45M 3S`, personally.

Reply:
Yeah I can do that